test/FakeStateMachine-run.ts

```diff
@@ -123,4 +124,60 @@ describe('FakeStateMachine#run()', () => {
       );
     });
   });
+
+  describe('when Pass state has undefined ResultPath', () => {
```

Comment:
Is it required?

Reply:
Not sure what happened here it looks like it's a side effect of #25. I've removed.